app/main/views/frameworks.py

```diff
@@ -85,17 +86,43 @@ def framework_dashboard(framework_slug):
     if countersigned_framework_agreement_exists_in_bucket(framework_slug, current_app.config['DM_AGREEMENTS_BUCKET']):
         countersigned_agreement_file = COUNTERSIGNED_AGREEMENT_FILENAME
 
+    application_made = supplier_is_on_framework or (len(complete_drafts) > 0 and declaration_status == 'complete')
```

Comment:
It looks like we've done a similar check before. @TheDoubleK do you remember if there's a reason to keep the draft checks here? Drafts could be removed in the future and I thought we're settings `onFramework` flag for all suppliers now.